ckan/logic/schema.py

```diff
@@ -411,6 +411,7 @@ def default_user_schema():
         'reset_key': [ignore],
         'activity_streams_email_notifications': [ignore_missing],
         'state': [ignore_missing],
+        'extras': [ignore_missing, dict]
```

Comment:
dict won't work as a validator because ValueErrors aren't caught.  Maybe a string_string_dict function  to check that we were passed a dict with unicode keys and values.